pipeline/metadata/schema.py

```diff
@@ -103,12 +106,12 @@ class ReceivedHttps:
   blockpage: Optional[bool] = None
```

Comment:
Can we rename this to `is_known_blockpage`? This name is confusing.

Reply:
https://github.com/censoredplanet/censoredplanet-analysis/pull/137